src/main/actions/file.js

```diff
@@ -96,7 +108,7 @@ ipcMain.on('AGANI::response-close-confirm', (e, { filename, pathname, markdown }
   const win = BrowserWindow.fromWebContents(e.sender)
   dialog.showMessageBox(win, {
     type: 'warning',
-    buttons: ['Save', 'Cancel', 'Delete'],
+    buttons: ['Save', 'Cancel', 'Exit and don\'t save'],
```

Comment:
I think `Delete` is the meaning of `Exit and don\'t save`, and many application use it, So I think user knew the exact meaning of `Delete`.

Reply:
Ok, I've never seen that before. I'll change that later.